packages/lib/README.md

```diff
@@ -0,0 +1,65 @@
+### `lib` Directory Readme
+
+#### Overview
+
+The `lib` directory is a new addition to our project structure, designed to
```

Comment:
[nit] I feel like when we use the word "new" in these docs it never gets deleted and just ends up out of date in a few months
```suggestion
The `lib` directory is designed to
```